scss/components/token.scss

```
@@ -0,0 +1,44 @@
+@import "./../settings";
+@import "./../mixins";
+@import "./../icons/mixins";
+@import "./../functions";
+
+///////////////////////////////////////////////////////////////////////////////
+/// .fd-tag has been depricated as it has been renamed to .fd-token
+/// this file and all realted artifacts for fd-tag should be removed
+/// after couple of releases. .fd-tag will continue to work in the meantime
+///////////////////////////////////////////////////////////////////////////////
+
+/*!
+.fd-tag+(--no-border)
+    .fd-tag__content+()
+    .fd-tag__title+()
+*/
+$block: #{$fd-namespace}-token;
+
+.#{$block} {
```

Just remove the `tag` files and just add the class here.

```suggestion
$block: #{$fd-namespace}-token;
$block-alt: #{$fd-namespace}-tag;

.#{$block},
.#{$block-alt} {
```

There's no need to output 2x the CSS.